components/proxy/Dockerfile

```diff
@@ -5,9 +5,23 @@ FROM $IMAGE_NAME:$IMAGE_VERSION
 LABEL maintainer="Quality-time team <quality-time@ictu.nl>"
 LABEL description="Quality-time proxy"
 
+ENV NGINX_LISTEN 80
+
 ENV FRONTEND_HOST frontend
 ENV FRONTEND_PORT 5000
+
 ENV API_SERVER_HOST api_server
 ENV API_SERVER_PORT 5001
 
-COPY *.template /etc/nginx/templates/
+RUN rm -f /etc/nginx/conf.d/default.conf
+RUN rm -f /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
+RUN rm -f /docker-entrypoint.d/20-envsubst-on-templates.sh
```

Comment:
Can this pre-existing script not provide what `90_fix_listen.sh` does?

Reply:
This file can be removed if we switch the 'LISTEN 80' statement to 'LISTEN 8080', we can remove also remove the 'user nginx' statement. Because user switching is only supported when Nginx is running with Root permissions.